Lib/test/test_enum.py

```diff
@@ -4440,6 +4440,14 @@ def test__all__(self):
 CONVERT_STRING_TEST_NAME_E = 5
 CONVERT_STRING_TEST_NAME_F = 5
 
+# We also need values that cannot be compared:
```

Comment:
All we need for this test is for `UNCOMPARABLE_*` to be different types of values:

`UNCOMPABLE_A` = 5
`UNCOMPABLE_C` = (9, 4)
`UNCOMPABLE_B` = 'hello'